test.env

```diff
@@ -1,5 +1,6 @@
 DB_HOST = localhost
 DB_USER = raquelmms
 DB_PASS = 123abc
-DB_NAME = raquelmms_test
-PORT= 4000
+DB_NAME = postgres
+PORT= 5433
+DATA_URL = postgres://raquelmms:123abc@localhost:5433/postgres
```

Comment:
Se vc for usar a URL pra conectar com o banco, então as outras variáveis de banco `DB_*` não seriam mais necessárias.